res_users_password_security/README.rst

```diff
@@ -0,0 +1,94 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Password  Security
+==================
+
+This module allows admin to set company-level password security requirements
+and enforces them on the user.
+
+It contains features such as
+
+* Password expiration days
+* Password length requirement
+* Password minimum number of lowercase letters
+* Password minimum number of uppercase letters
+* Password minimum number of numbers
+* Password minimum number of special characters
+
+Configuration
+=============
+
+* Navigate to company you would like to set requirements on
+* Click the ``Password Policy`` page
+* Set the policies to your liking.
+
+Password complexity requirements will be enforced upon next password change for
+any user in that company.
+
+
+Settings & Defaults
+-------------------
+
+These are defined at the company level:
+
++---------------------+---------+------------------------------------------+
```

Comment:
just a note: why dont use simple rst table format? much easier to handle and maintain. In your case you dont need the complex format